src/python/sortingview/gui/extensions/correlograms/CorrelogramSubsamplingInfo.md

```diff
@@ -0,0 +1,13 @@
+# Subsampling
+
+For efficiency, sortings are subsampled when creating auto- and cross-correlograms.
+Sampling using tight time windows poses the risk that one of a pair of firing events
```

Comment:
I'd instead say something like:

Pure random subsampling of events is not optimal because it is unlikely that the spikes in a event pair appearing in the histogram will both be in the subsampling. An alternative is to sample in a long contiguous time segment. However, this induces bias toward a particular part of the recording.
...